### PR TITLE
Interview questions

### PR DESCRIPTION
**Pull request by lfunderburk**
I am trying a new format this week
I will be asking questions in an interview format on LinkedIn, and sharing an interactive notebook on binder for people to participate using JupySQL

The 'post' notebook is a notebook I am using to generate the questions and answers using OpenAI. This can be ignored.

Closes https://github.com/ploomber/posts/issues/44

### CODE REVIEW

**Comment by lfunderburk** [comment]
Sample post

> 🎯 SQL Bootcamp Day 1! 🚀
> 
> Today, we're diving into the intriguing 'absenteeism at work' dataset. This dataset includes detailed records of employee absenteeism, and includes factors such as reason for absence, day of the week, seasons, distance from residence to work, age, and many more.
> 
> Question 1 (Easy): Let's start with a warm-up. Can you tell us how many records are there in the 'absenteeism' table? This is a great place to start to get a sense of the scale of the data we are dealing with.
> 
> Question 2 (Medium): Let's crank it up a notch. Can you find out on which days of the week the average absenteeism time exceeds 4 hours? This question will require you to use aggregate functions and to group your data by the day of the week.
> 
> Question 3 (Hard): Now let's dive into the deep end. Can you find the top 3 ages with the highest total absenteeism hours, excluding disciplinary failures? This will require you to filter your data, aggregate it, and then order it to find the top 3 ages.
> 
> Try out these questions in the interactive notebook here: https://buff.ly/3HmVTtt
> 
> Don't forget to share your answers, thoughts, and questions in the comments below! If you find these questions helpful or fun, please give it a like or share it with your network. Let's help each other grow and learn together!
> 
> Stay tuned for more challenges in the coming days. Happy querying!
> 
> #SQL #DataAnalysis #DataChallenge #LearningTogether #SQLBootcamp #DataCommunity

**Comment by lfunderburk** [comment]
Moving to https://github.com/ploomber/projects

**Comment by lfunderburk** [comment]
Hi Ido, 

Thank you for chatting more about this with me. 

I consolidated a "format" - this format is what is followed in each of the notebooks for day 1, 2, 3, 4. 

It can be found in this PR, currently in the README file

Linked here

https://github.com/ploomber/posts/blob/83c4cc1e54fc691a72be733a29ba9e5850f2f9e4/interview-questions-posts/README.md

You can make suggestions on this file to make it easier for you to review

File: interview-questions-posts/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+from sqlalchemy.engine import create_engine
+
+url = "https://archive.ics.uci.edu/ml/machine-learning-databases/00445/Absenteeism_at_work_AAA.zip"
+
+# download the ZIP file
+response = requests.get(url)
+
+# extract the contents of the ZIP file
+zf = zipfile.ZipFile(io.BytesIO(response.content))
+df = pd.read_csv(zf.open("Absenteeism_at_work.csv"), sep=";", index_col=0)
```
[the remark this answers is not in the document]
Comment (lfunderburk) [reply interview-questions-posts/README.md]: Resolved in latest commit

File: interview-questions-posts/day_1.ipynb [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<details>\n",
+    "\n",
+    "<summary>Answers</summary>\n",
```
[the remark this answers is not in the document]
Comment (lfunderburk) [reply interview-questions-posts/day_1.ipynb:202]: The details tag makes the collapsed by default - there is a left arrow to collapse and hide

File: interview-questions-posts/day_1.ipynb [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### References   \n",
+    "\n",
+    "Martiniano, A., Ferreira, R. P., Sassi, R. J., & Affonso, C. (2012). Application of a neuro fuzzy network in prediction of absenteeism at work. In Information Systems and Technologies (CISTI), 7th Iberian Conference on (pp. 1-4). IEEE.\n",
+    "\n",
+    "### Acknowledgements\n",
```
[the remark this answers is not in the document]
Comment (lfunderburk) [reply interview-questions-posts/day_1.ipynb]: Added more resources and linked Mark's tutorial

**Comment by lfunderburk** [comment]
Hello Ido, 

Thank you for feedback. Latest changes with your suggestions incorporated. 

Additions:
- Marks tutorial
- Replaced SQLite with duckdb
- Added more resources (linked documentation to visualization and large queries tutorial)

Hoping it's looking ready to go - if yes I will blast these next week

https://github.com/ploomber/posts/blob/b75dcd29e6b4867aea33591e9672ad6450b741d6/interview-questions-posts/README.md

**Review by edublancas — CHANGES_REQUESTED** [review]
overall, it looks good. let's keep this content here, this is the right repo (the projects repo is for ploomber examples).

let's just add some context paragraph to the README.md, from what I see this is content that will be posted on socials over the next few days so let's ensure there is some intro text describing that this is so future readers know

**Comment by lfunderburk** [comment]
Thank you Eduardo

Added more context to the readme

**Comment by lfunderburk** [comment]
Once it's merged I will schedule the posts for this week and next

I will continue generating notebooks. Might be worth chatting a bit on the workflow (i.e. creation, review, publishing)

What I am thinking is each PR should be data-set centric.

**Pull request merged**